cmd/ddev/cmd/config_test.go

```diff
@@ -470,11 +470,20 @@ func TestCmdDisasterConfig(t *testing.T) {
 	err = os.Chdir(subdir)
 	assert.NoError(err)
 
-	// Make sure that ddev config in a subdir gives an error
+	// Make sure that 'ddev config' in a subdir doesn't give an error for existing project
```

Comment:
TestCmdDisasterConfig needs a better name (and comment). OK to rename. It was built around the homedir failure of course, and now it's covering more ground.